Documentation/dev/debugging.md

```diff
@@ -169,3 +170,34 @@ kubectl -n $METERING_NAMESPACE port-forward hdfs-datanode-0 9864
 ```
 
 To check other datanodes, run the above command, replacing `hdfs-datanode-0` with the pod you want to view information on.
+
+## Metering Ansible Operator
+Metering uses the ansible operator to watch and reconcile resources in a cluster environment. When debugging a failed Metering install, it can be helpful to view the Ansible logs or status of your `MeteringConfig` custom resource.
+
+##### Accessing Ansible Logs
+There are a couple of ways of accessing the Ansible logs depending on how you installed the Metering resources.
+
+In a typical install, the Metering operator is deployed as a pod. In this case, we can simply check the logs of the `ansible` container within this pod:
+```
+kubectl -n $METERING_NAMESPACE logs $(oc get pods -l app=metering-operator -o name | cut -c 5-) -c ansible
+```
+
+Alternatively, you can view the logs of the `operator` container (replace `-c ansible` with `-c operator`) for condensed output.
+
+If you are running the Metering operator locally (i.e. `make run-metering-operator-local`), there won't be a dedicated pod, and so you would need to run the following command ():
+```
+docker exec -it metering-operator bash -c 'tail -n +1 -f /tmp/ansible-operator/runner/metering.openshift.io/v1/MeteringConfig/*/*/artifacts/*/stdout'
+```
+
+When tracking down a failed task, you may encounter this output:
+```
+changed: [localhost] => (item=None) => {"censored": "the output has been hidden due to the fact that 'no_log: true' was specified for this result", "changed": true}
+```
+
+This is because we specify the Ansible property, `no_log: true`, on output-extensive tasks (running helm template, creation of resources, etc.). If you fail on one of these tasks, you can set `no_log: false` on the respective task and re-run for more information on why your installation failed.
```

Comment:
We actually only have a few tasks like this, and we have an option to enable logging the helm template task output now, directly from the MeteringConfig CR: `spec.logHelmTemplate=true`

Reply:
yeah, pretty much forgot about that field - that's a better suggestion.

Reply:
It's new.